app/components/channel_list/header/plus_menu/index.tsx

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {useIntl} from 'react-intl';
+import {DeviceEventEmitter} from 'react-native';
+
+import CompassIcon from '@components/compass_icon';
+import {Events, Screens} from '@constants';
+import {useTheme} from '@context/theme';
+import {showModal} from '@screens/navigation';
+import EphemeralStore from '@store/ephemeral_store';
+
+import PlusMenuItem from './item';
+
+const PlusMenuList = () => {
+    const intl = useIntl();
+    const theme = useTheme();
+
+    const browseChannels = useCallback(async () => {
+        DeviceEventEmitter.emit(Events.CLOSE_BOTTOM_SHEET);
+        await EphemeralStore.waitUntilScreensIsRemoved(Screens.BOTTOM_SHEET);
+
+        const title = intl.formatMessage({id: 'browse_channels.title', defaultMessage: 'More Channels'});
+        const closeButton = await CompassIcon.getImageSource('close', 24, theme.sidebarHeaderTextColor);
+
+        showModal(Screens.BROWSE_CHANNELS, title, {
+            closeButton,
+        });
+    }, [intl, theme]);
+
+    const createNewChannel = useCallback(async () => {
+        // To be added
+    }, [intl, theme]);
+
+    const openDirectMessage = useCallback(async () => {
+        // To be added
+    }, [intl, theme]);
```

Comment:
As I said before, almost sure the create new channel is not for all users. Not sure about openDirectMessage with guests.

Reply:
given there is no content on the function, we can check for that later

Reply:
All guests can open direct channels, the caveat is that they can only open DM's with the people that share a channel with. That should be handled in the open/create DM/GM screen. Basically that needs to be handled in this PR https://github.com/mattermost/mattermost-mobile/pull/5900